hosted/gke/p0/p0_suite_test.go

```diff
@@ -61,3 +60,85 @@ var _ = ReportAfterEach(func(report SpecReport) {
 	// Add result in Qase if asked
 	Qase(testCaseID, report)
 })
+
+func p0upgradeK8sVersionCheck(cluster *management.Cluster, client *rancher.Client, clusterName string) {
+	currentVersion := cluster.GKEConfig.KubernetesVersion
+	versions, err := helper.ListGKEAvailableVersions(client, cluster.ID)
+	Expect(err).To(BeNil())
+	Expect(versions).ToNot(BeEmpty())
+	upgradeToVersion := &versions[0]
+
```

Comment:
GinkgoLogr.Info(fmt.Sprintf("Upgrading cluster to GKE version %s", upgradeToVersion))